src/pretalx/htmlexport/__init__.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class HtmlExportConfig(AppConfig):
+    name = 'pretalx.htmlexport'
```

Comment:
Hm, it might be better to call the app ``export``, since I'd expect other export formats to go there, as well. Alternatively, I could also see all things you put in here in the ``agenda`` app with no actual need for another app.

Reply:
moved. :)